### DISCUSSION TITLE
2024 Academic conference workshop, 2024-06-28 강릉

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#429] [Workshop]
안녕하세요.

2024 Academic conference 진행을 했던 김종필입니다.

이번 달 6월 28일 금요일 오후 4시 30분에 workshop을 진행합니다.
계속 모임에 참여하셨던 분들에게는 대략적으로 설명 드렸는데
날짜와 장소를 공지해 드립니다.

식비는 제가 부담하는 워크숍으로 나머지 비용만 부담하시면 되는 여행 모임입니다.
아래 일정은 제가 제안 드리는 일정으로
개인적으로 다른 일정이 있으신 분은 따로 움직이셔도 됩니다.

### 이동 장소에 대한 지도 요약

- 각 장소당 도보 이동 시간은 10분 내외입니다.
- 위치를 잘 보시고 적당한 곳에 숙소를 잡으시면 됩니다.

<img width="1013" alt="image" src="https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/054c3ae5-4415-4cba-8997-33b94ccc6198">

#### 오후 4시 30분, 더그라운드

- https://naver.me/5aqQHkSS
- 모임 시간이 4시 30분인 이유는 4시 30분 까지 런치세트를 팔기 때문입니다.
- (default) 메뉴 주문은 기네스 생맥주 + 코코넛 슈림프 튀김입니다.
- 2024 academic conference 책 3종 논의 및 결정
  - https://github.com/ThinkAboutSoftware/AcademicConference/discussions/428
- 가게 앞에서 강문 해변까지 1분도 안걸리므로 강문 해변 배경으로 단체 사진

#### 오후 7시, 강릉해파랑물회

- https://naver.me/xY9854AT
- (default) 해파랑 물회 세트 + 오징어순대 조합으로 먹어볼 예정입니다.

#### 오후 9시 전후

- 역사와 전통을 자랑하는 워크샵 불꽃놀이
- 올해는 소소하게 스파클라 불꽃놀이만 할 예정입니다.

#### 오후 12시, 강릉짬뽕순두부 동화가든 본점

- https://naver.me/GLS1BVmD
- 주말 점심시간 대기예약 30분 이상 예상되므로, 제가 미리 가서 대기 예약을 걸어 놓을 예정입니다.
- 대부분 숙소 체크아웃이 11시 이므로, 12시 전에 모이는 걸로 하면 대략 맞을 것으로 예상합니다.
- (default) 원조짬순(짬뽕순두부)

#### 오후 1시, 강릉당커피콩빵 강문해변점

- https://naver.me/Fn2ivKre
- (default) 아메리카노 아이스 + 커피콩빵 1개
- (option, 강릉 다녀왔다고 자랑할 수 있는) 포장용 커피콩빵도 사 드릴 예정입니다.

#### 오후 1시 30분, 강문해변에서 해산

---

추가 TMI로 제 개인 일정입니다.

### 교통편

- 6월 28일 금요일, 서울 13:31 - 강릉 15:23, KTX-이음 813 
- 6월 29일 토요일, 강릉 15:24 - 서울 17:27, KTX-이음 816

### 숙소

- 세인트존스호텔, 오션뷰 더블룸

### 강릉 숙소 근처 이동수단

개인 접이식 미니벨로 자전거를 가지고 와서
강릉역에서 숙소, 해산 후 강릉역 이동할 때 탈 예정입니다.
또 토요일 일정 이후 해변 근처 자전거 라이딩 예정입니다.

**Comment by yeslee-v** [comment]
~~이거 바라보고 북클럽 신청했었는데(!) 그 다음 날 오전 일찍 일정이 있어 참여가 어렵게 됐습니다ㅠㅠ~~
보류입니다!!!!!!!!!!!!!

**Comment by jongfeel** [comment]
## 강릉 워크샵 사진들

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/830d708c-fa86-48dc-87fb-150fb095dc44)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/ceccb8a6-8385-485a-979a-cd1574ce4d26)

https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/8979c1e4-b60a-421a-9674-eecf1f936bce

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/23e56b20-36ff-4642-95a6-72b4de7416c8)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/959c011b-f832-4ca8-a965-3f86acb5123b)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/e2aaea10-cd32-4872-8a35-7d9afb74ae56)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/c28f29e4-563f-4983-a4b9-90d34400bde3)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/95a96ffb-d3dd-4013-ae45-6fbcfb772f63)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/92da931b-69d6-4997-be3c-3b9e56628d7a)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/52b3b341-9d51-4b5e-b8e7-bdc1aef5368f)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/33e226fc-9b2a-40c6-bb03-df51d7b02755)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/82b58b82-b82f-463b-b849-abc03d88da36)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/9f85b2fe-3bc3-44f3-b97c-15f1618ea0af)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/42a9cc10-f036-43be-9f11-6a748a3aee57)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/379eca70-f215-42c1-b56d-deb2ce1d45f9)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/3c1ed28b-07a6-4a7a-af7e-375fab20daf7)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/b580ba1a-67a3-4fe3-a95e-ec134df3bd14)
